DUI3-DX/Converters/Revit/Speckle.Converters.RevitShared/ToSpeckle/FamilyInstanceConversionToSpeckle.cs

```diff
@@ -0,0 +1,143 @@
+using Autodesk.Revit.DB;
+using Autodesk.Revit.DB.Structure;
+using Speckle.Converters.Common.Objects;
+using Speckle.Converters.RevitShared.Extensions;
+using Speckle.Converters.RevitShared.Helpers;
+using Speckle.Core.Models;
+
+namespace Speckle.Converters.RevitShared.ToSpeckle;
+
+public sealed class FamilyInstanceConversionToSpeckle : BaseConversionToSpeckle<DB.FamilyInstance, Base>
+{
+  private readonly IRawConversion<DB.Element> _elementConverter;
+  private readonly IRawConversion<DB.FamilyInstance, SOBR.RevitBeam> _beamConverter;
+  private readonly IRawConversion<DB.FamilyInstance, SOBR.RevitColumn> _columnConverter;
+
+  public FamilyInstanceConversionToSpeckle(
+    IRawConversion<Element> elementConverter,
+    IRawConversion<FamilyInstance, SOBR.RevitBeam> beamConverter,
+    IRawConversion<FamilyInstance, SOBR.RevitColumn> columnConverter
+  )
+  {
+    _elementConverter = elementConverter;
+    _beamConverter = beamConverter;
+    _columnConverter = columnConverter;
+  }
+
+  public override Base RawConvert(FamilyInstance target)
+  {
+    Base? @base = null;
+
+    ////adaptive components
```

Comment:
Commented out section

Reply:
I wonder if some of these is a bit messy and we need a more specific conversion to avoid all of the code paths